src/appMain/smartDeviceLink.ini

```diff
@@ -86,7 +86,7 @@ TimeTestingPort = 8090
 ReadDIDRequest = 5, 1
 ; Limitation for a number of GetVehicleData requests (the 1st value) per (the 2nd value) seconds
 GetVehicleDataRequest = 5, 1
```

Comment:
we shall have a similar rate limitation of GetInteriorVehicleData request, 
GetInteriorVehicleDataRequest = 5, 1